tests/dummy/app/pods/application/template.hbs

```diff
@@ -15,8 +15,11 @@
       <div class='frost-modal-demo-selector-title'>
         Cookbook
         {{#link-to 'simple'}}Basic{{/link-to}}
+        {{#link-to 'simple-loading'}}Basic - Loading{{/link-to}}
```

Comment:
@sglanzer The demo app currently uses small viewports for the existing demos so that users don't have to scroll as far to experience the infinite paging experience, not as many records need to be created for the paging screens, etc.   Because this viewport is not very tall (due to it's overflow being hidden) the loading indicator does not show in its entirety.  In the new routes I created to demo the loading support I have increased the viewport size.  If you have any recommendations on how to remedy this or approach it differently I am interested.  In either case I am in need of direction on how you wish me to proceed.